packages/forma-36-react-components/src/components/Dropdown/README.mdx

```diff
@@ -38,7 +38,7 @@ function DropdownExample() {
       toggleElement={
         <Button
           size="small"
-          buttonType="muted"
+          variant="transparent"
```

Comment:
```suggestion
          variant="secondary"
```

I think the variant `muted` is now called `secondary`

The transparent one is the old "naked" button and if I remember correctly we wanted to recommend the use of secondary buttons instead of the naked ones

Am I remembering this correctly or dreaming @fabe and @domarku?

maybe I'm wrong, but my problem with transparent buttons is that they don't differentiate themselves from the rest of the UI enough

![Screenshot 2021-08-12 at 17 51 52](https://user-images.githubusercontent.com/6597467/129228348-fd3a2f85-5fa5-4ca5-8255-4152b05107ba.png)

Reply:
@gui-santos we decided to re-implement naked version but we called it "transparent" now. It is needed for cases, like `close` IconButton (now it will be handled by button component). So we actually need this variant.

Reply:
ok, but it's being used in places that are not `IconButton` in the PR

Like in the screenshot I posted, I believe those buttons should be `primary` or `secondary` I know it's just documentation, but I think we want to improve the UX of the docs as well 

![Screenshot 2021-08-13 at 13 12 16](https://user-images.githubusercontent.com/6597467/129349049-f9ed6a1c-48f4-401b-b5ee-1ba72bbc3034.png)

I understand why the `IconButton` uses transparent variant, but the IconButton feels "clickable" hahahah
I guess we just wanted to use all the Button features in the IconButton without the border, but for normal buttons we fall in the same problem we had with the `naked` variant. When it's used alone, without other buttons, the naked/transparent button is just text that the user needs to guess if it's clickable or not 🤷‍♂️

Reply:
Got it, rolled back to "secondary" here.

Reply:
You forgot this one

```suggestion
          variant="secondary"
```